src/components/ha-analytics.ts

```diff
@@ -0,0 +1,128 @@
+    if (!this.analytics.huuid) {
+      return html``;
+    }
+    return html`
+      <p>
+        ${this.hass.localize(
+          "ui.panel.config.core.section.core.analytics.instance_id",
+          "huuid",
+          this.analytics.huuid
+        )}
+      </p>
+      ${PREFERENCES.map((preference) =>
+        preference === "base" || this.analytics.preferences.includes("base")
+          ? html`<ha-settings-row>
+              <span slot="prefix">
+                <ha-checkbox
+                  @change=${this._handleRowCheckboxClick}
+                  .checked=${this.analytics.preferences.includes(preference)}
+                  .preference=${preference}
+                >
+                </ha-checkbox>
+              </span>
+              <span slot="heading">
+                ${preference === "usage"
+                  ? isComponentLoaded(this.hass, "hassio")
+                    ? this.hass.localize(
+                        `ui.panel.config.core.section.core.analytics.preference.usage_supervisor.title`
+                      )
+                    : this.hass.localize(
+                        `ui.panel.config.core.section.core.analytics.preference.usage.title`
+                      )
+                  : this.hass.localize(
+                      `ui.panel.config.core.section.core.analytics.preference.${preference}.title`
+                    )}
+              </span>
+              <span slot="description">
+                ${preference === "usage"
+                  ? isComponentLoaded(this.hass, "hassio")
+                    ? this.hass.localize(
+                        `ui.panel.config.core.section.core.analytics.preference.usage_supervisor.description`
+                      )
+                    : this.hass.localize(
+                        `ui.panel.config.core.section.core.analytics.preference.usage.description`
+                      )
+                  : this.hass.localize(
+                      `ui.panel.config.core.section.core.analytics.preference.${preference}.description`
+                    )}
+              </span>
+            </ha-settings-row>`
+          : ""
+      )}
+      <p>
+        ${this.hass.localize(
+          "ui.panel.config.core.section.core.analytics.documentation",
+          "link",
+          html`<a href="https://www.home-assistant.io/integrations/analytics">
+            https://www.home-assistant.io/integrations/analytics
+          </a>`
+        )}
+      </p>
+    `;
+  }
+
+  private _handleRowCheckboxClick(ev: Event) {
+    const checkbox = ev.currentTarget as HaCheckbox;
+    const preference = (checkbox as any).preference;
+    let preferences = this.analytics.preferences;
+
+    if (checkbox.checked) {
+      if (preferences.includes(preference)) {
+        return;
+      }
+      preferences.push(preference);
+    } else {
+      preferences = preferences.filter((entry) => entry !== preference);
+    }
+
+    this.analytics = { ...this.analytics, preferences };
```

Comment:
we should not change the data we get from the parent, but instead, fire an event with a copied new version